aspnet/conceptual-overview/aspnet.rst

```diff
@@ -42,7 +47,20 @@ In summary, with ASP.NET Core you gain the following foundational improvements:
 Application anatomy
 -------------------
 
-ASP.NET Core applications are defined using a public ``Startup`` class:
+.. comment In RC1, The work of the WebHostBuilder was hidden in dnx.exe
+
+An ASP.NET Core app self-hosts in a console app using ``IWebHost``. 
+
+.. literalinclude:: /getting-started/sample/aspnetcoreapp/Program.cs
+    :language: c#
+    
+The ASP.NET Core app is simply a console app that creates a web server in its ``Main`` method. ``Main`` uses `WebHostBuilder <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Hosting/WebHostBuilder/index.html>`__ , which follows the builder pattern, to create a hosting engine for ``IWebHost`` and to build the HTTP pipeline. The builder has methods that define the web server type (for example ``UseKestrel``) and the startup class (``UseStartup``). In the example above, the Kestrel web server is used, but other web servers can be specified. We'll show more about ``UseStartup`` in the next section. ``WebHostBuilder`` provides many optional methods including ``UseIISIntegration`` for hosting in IIS and IIS Express and ``UseContentRoot`` for specifying the root content directory. The ``Build`` and ``Run`` methods build the ``IWebHost`` that will host the app and start it listening for incoming HTTP requests.
+
+
+Startup
+---------------------------
+
+The ``Startup`` class (``UseStartup<Startup>`` in the code below) must be public and contain the following methods:
```

Comment:
No need to dive into these details about the Startup class here - this content is already in the [Application Startup](https://docs.asp.net/en/latest/fundamentals/startup.html) article and you should just link to it. Instead say that the Startup class is where the developer defines the request handling pipeline for the application and also where any services needed by the application are configured.